File: openapi/code_samples/PHP/branches/create.php
<?php
$client = new Client();
$headers = [
  'Authorization' => 'eyJhbGciOiJIUzUxMiJ9.eyJyb2xlIjoidmVuZG9yIiwidXNlcm5hbWUiOiJ2X2F4ZWxfOSIsImlhdCI6MTY1ODE2NDM4OCwiZXhwIjoxNjU4NzY5MTg4fQ.vvLtYB4O73futNvwzTZt2LhFzNV--XvMfTT5-62YWtxX-9CY6X4-UtIVDkCR_MX4YqX83AcCXiquHAb8uNHIjw',
  'Content-Type' => 'application/json'
];
$body = '{
  "name": "string",
  "idMunicipality": 0,
  "addressGeo": "string",
  "addressHuman": "string",
  "latitude": 0,
  "longitude": 0,
  "active": true,
  "vacation": true
}';
$request = new Request('POST', 'https://api.tizo.co/api/v1/vendors/stores/:idStore/branches/', $headers, $body);
$res = $client->sendAsync($request)->wait();
echo $res->getBody();

File: openapi/code_samples/python/store/create.py
import requests
import json

url = "https://api.tizo.co/api/v1/vendors/stores/"

payload = json.dumps({
  "name": "TADA",
  "ruc": "12121",
  "email": "agarciadarce@gmail.com",
  "dba": "AA",
  "isPrivate": True,
  "pathImage": "",
  "phone": "+50587878787",
  "description": "AA",
  "vacation": False
})
headers = {
  'Authorization': 'eyJhbGciOiJIUzUxMiJ9.eyJyb2xlIjoidmVuZG9yIiwidXNlcm5hbWUiOiJ2X2dhYnJpZWxfNiIsImlhdCI6MTY1NzgwOTQ2MSwiZXhwIjoxNjU4NDE0MjYxfQ.N_rPvlIFgpj_mK8mcWzDbKEgud6bJeSNc5cOKqzyYTvWeCqStlzUCQjeyoNt98GwqpEU-4jdI35jMk83bahzpg',
  'Content-Type': 'application/json'
}

response = requests.request("POST", url, headers=headers, data=payload)

print(response.text)
